packages/features/ee/organizations/components/CreateANewOrganizationForm.tsx

```diff
@@ -196,6 +198,25 @@ export const CreateANewOrganizationForm = ({ slug }: { slug?: string }) => {
           />
         </div>
 
+        <div className="mb-5">
+          <Controller
+            name="isPlatform"
+            control={newOrganizationFormMethods.control}
+            defaultValue={false}
+            render={({ field: { value } }) => (
+              <div className="[&_label]:mt-1">
+                <SettingsToggle
+                  title={t("organization_is_platform")}
+                  checked={value}
+                  onCheckedChange={(checked) => {
+                    newOrganizationFormMethods.setValue("isPlatform", checked);
+                  }}
+                />
+              </div>
+            )}
+          />
+        </div>
```

Comment:
is this for admins only? if yes, it should be wrapped in isAdmin

if this is for self-serve too, we should instead build a self-serve checkout for platform customerss

Reply:
https://app.cal.com/settings/platform/new can reuse some of the same UI but is more tailored towards platform inputs